src/plugins/intel_cpu/src/nodes/detection_output.cpp

```diff
@@ -130,7 +130,7 @@ void DetectionOutput::prepareParams() {
     int cacheSizeL3 = utils::get_cache_size(3, true);
     isSparsityWorthwhile =
         (confidenceThreshold > sparsityThreshold) &&
-        ((classesNum * priorsNum * sizeof(float) * 2) > cacheSizeL3);
+        ((classesNum * priorsNum * sizeof(float) * 2) > static_cast<size_t>(cacheSizeL3));
```

Comment:
Yes, "cacheSizeL3" change to size_t is more reasonable. 
But it has just from "int cacheSizeL3 = utils::get_cache_size(3, true);", utils::get_cache_size is used in many places.
So I also think this update have minimal modifications.

Reply:
This func is used to many places, my initial idea is we should have mininal changes.
I just check `utils::get_cache_size`, it will call `dnnl::impl::cpu::platform::get_per_core_cache_size`, it will return `unsigned`, so now it seams to fixing `utils::get_cache_size` will be good idea.